player:                         # General parameters for motorolla eval board interface
    network_ip: 192.168.0.70            
    network_port: 10001
motors:
    drive:
        max_speed: 1.0      # 1.53 m/s
        min_speed: 0.1      # rad/s
        motor_direction: -1
    flipplers:
        max_speed: 1.0
        min_speed: 0.1
        motor_direction: -1      
sensors:
    gps:
        topic: jaguar_gps
        ip: 192.168.0.71
        port: 10002
    imu:
        topic: jaguar_imu
        ip: 192.168.0.71
        port: 10001
robot_control_topics:
    drive_control: cmd_vel
    front_flipper_control: front_flipper_cmds
    rear_flipper_control: rear_flipper_cmds
    headlight_control: headlight_cmds
